src/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -1139,6 +1139,25 @@ void fightBombingRaids(final IDelegateBridge delegateBridge, Supplier<Collection
     }
   }
 
+  public void fightAutoKills(final IDelegateBridge delegateBridge) {
+    // Kill undefended transports. Done here to remove potentially dependent sea battles below
+    for( final Territory t : getPendingBattleSites(false) ) {  // Loop through normal combats i.e. not bombing or air raid
```

Comment:
Not sure what you mean. Lambda's are largely beyond my basic knowledge of Java.

I don't have eclipse on this computer and <1GB of disk space so might not be able to fit it. 

I've updated the comments to fit within 120 characters though.